### DISCUSSION TITLE
`reactpy-dashboard`: Admin panel/dashboard for your next project

### DISCUSSION THREAD

**Discussion by stevej2608** [idom-team/idom#1220] [Downloadable Packages]
[reactpy-dashboard](https://github.com/stevej2608/reactpy-dashboard) is a 100% python/reactpy based clone of the  beautiful of the Themesberg [tailwind-dashboard-windster](https://demo.themesberg.com/windster/) dashboard.

- [X] Tailwind CSS
- [X] Modular forms
- [X] Apex charts
- [X] Tables with pagination/search/sort
- [X] Dark/Light mode switch

**Comment by hasansezertasan** [comment]
Looks promising. I'd like to see a demo app if possible.

Great job 🚀🚀🚀.
